### DISCUSSION TITLE
[43160] Metro Last Light

### DISCUSSION THREAD

**Discussion by NineSnine** [pjy612/SteamManifestCache#65] [Share]
appid:43160
keys:43161;
aaa7d8fa6a079dfa9f4539d14ad5d597ae9eaea8d1b47df83d83e7b5b3735f73
keys:43162;
049f4d43c02d5084bb366177de273ba7a8e1dce19581d7bd3af8fe888398d6dc
code:71D098043D37C363
email:[w424334068@163.com]
[metro last light.zip](https://github.com/pjy612/SteamManifestCache/files/13694569/metro.last.light.zip)

**Comment by pjy612** [comment]
查收
